### PR TITLE
A few minor yet necessary changes in readme file

### PR DESCRIPTION
**Pull request by TheSoumilArora**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # transfer.sh [![Go Report Card](https://goreportcard.com/badge/github.com/dutchcoders/transfer.sh)](https://goreportcard.com/report/github.com/dutchcoders/transfer.sh) [![Docker pulls](https://img.shields.io/docker/pulls/dutchcoders/transfer.sh.svg)](https://hub.docker.com/r/dutchcoders/transfer.sh/) [![Build Status](https://github.com/dutchcoders/transfer.sh/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/dutchcoders/transfer.sh/actions/workflows/test.yml?query=branch%3Amaster)
 
-Easy and fast file sharing from the command-line. This code contains the server with everything you need to create your own instance.
+Easy and fast file sharing from the command-line. This code contains the server alongwith everything you need to create your own instance.
```
[the remark this answers is not in the document]
Comment (whosoumilarora) [reply README.md:3]: yes it is now correct in the main file

[the remark this answers is not in the document]
Comment (aspacca) [reply README.md:3]: still not corrected

[the remark this answers is not in the document]
Comment (whosoumilarora) [reply README.md:3]: when im downloading the readme file from my forked repository, it has both of these changes

**Comment by stefanbenten** [comment]
@whosoumilarora The comment from @aspacca is still valid and needs to be addressed before this can be merged. Would you mind looking into this?

**Comment by stefanbenten** [comment]
Closing, since stale and author is MIA.

**Pull request closed without merging**